client/src/app/main/dashboard/dashboard-edit-profile/dashboard-edit-profile-page.js

```diff
@@ -118,7 +126,7 @@ class DashboardEditProfilePage extends React.Component {
             if(customField.type === 'select') {
                 parsedFrom[`customfield_${customField.name}`] = customField.options[form[customField.name]].name;
             } else {
-                parsedFrom[`customfield_${customField.name}`] = form[customField.name];
+                parsedFrom[`customfield_${customField.name}`.replace(/ /g,'_')] = form[customField.name];
```

Comment:
@LautaroCesso maybe better to create `getCustomFieldParamName($customFieldName: string)` function in libs and use it every time we need to do `customfield_${customField.name}`.replace(/ /g,'_')`, changing it to `getCustomFieldParamName(customField.name)`